store/store.go

```diff
@@ -1124,7 +1147,7 @@ var download = func(name, sha3_384, downloadURL string, user *auth.UserState, s
 		// connections) that led us to an error (the default client is
 		// documented as not reusing the transport unless the body is
 		// read to EOF and closed, so this is a belt-and-braces thing).
-		r, err := s.doRequest(&http.Client{}, reqOptions, user)
+		r, err := s.doRequest(newHTTPClient(0), reqOptions, user)
```

Comment:
this means using a logging transport for download, which means that if the transport gets `debugBody` we'll now have gigabytes worth of debug logs.
Easy fix though: in `store/logger.go`, add a content-type arg to the `debugBody()` method, and only return true if the content type is not `application/octet-stream`. Pass in the request/response content type when calling it from `RoundTrip`.
I'll poke you on telegram to see what your load is like; I can do these changes in a branch of my own.